changelog_entry.yaml

```
@@ -0,0 +1,4 @@
+bump: patch
```

__issue, minor, blocking:__ This needs to be a list item 
```suggestion
- bump: patch
```